homeassistant/components/solaredge/const.py

```diff
@@ -51,7 +51,7 @@
         json_key="lastYearData",
         name="Energy this year",
         entity_registry_enabled_default=False,
-        icon="mdi:solar-power",
```

Comment:
Why removing the icon? It's a specific icon that matches perfect here.

Reply:
We allow for exceptions in some cases, there is no need to drop the icon.
Also, this change is not related to this PR, so should be reverted.